multi_sqs_listener/_long_polling.py

```diff
@@ -27,11 +27,13 @@ def __init__(self, thread_id, queue_name, outbound_bus, run_event,
         Thread.__init__(self, name=queue_name)
         self.thread_id = thread_id
         self._region_name = kwargs['region_name'] if 'region_name' in kwargs else 'eu-west-1'
+        self._queue_acct_id = kwargs.get('queue_acct_id')
         self._outbound_bus = outbound_bus
         self._queue_name = queue_name
         self._handler_available_event = handler_available_event
         sqs = boto3.resource('sqs', region_name=self._region_name)
-        self._queue = sqs.get_queue_by_name(QueueName=self._queue_name)
+        self._queue = sqs.get_queue_by_name(QueueName=self._queue_name, QueueOwnerAWSAccountId=self._queue_acct_id)
```

Comment:
Thanks for this PR 👍
Could you ensure this still works with the current user account when no `_queue_acct_id` is provided?

Reply:
Oops, I've updated the PR to take this into account

Reply:
👍